rustls/src/server/handy.rs

```diff
@@ -46,32 +45,51 @@ mod cache {
         /// Make a new ServerSessionMemoryCache.  `size` is the maximum
         /// number of stored sessions, and may be rounded-up for
         /// efficiency.
+        #[cfg(feature = "std")]
```

Comment:
I would probably squash these changes into the previous commit.

Reply:
Done